client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,57 @@
+Updating Endpoint with Config File
+==================================
+
+.. TODO: link to Endpoint.update tutorial
+
+Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to cram everything in the Python script or the CLI command.
+
+For example:
+
+.. code-block:: json
+
+        {
+            "run_id": <experiment run id>,
+            "strategy": "canary",
+            "canary_strategy": {
+                "progress_step": 0.05,
+                "progress_interval_seconds": 30,
+                "rules": [
+                    {"rule": "latency_avg_max",
+                     "rule_parameters": [
+                         {"name": "threshold",
+                          "value": "0.1"}
+                    ]},
+                    {"rule": "error_4xx_rate",
+                     "rule_parameters": [
+                        {"name": "threshold",
+                         "value": "1"}
+                    ]}
+                ]
+            }
+        }
+
+This allows for easier inspection and modification later on.
+
+Verta's Client and CLI support updating Endpoint with JSON or YAML config file. This tutorial will show how it can be done, using the Client and CLI.
+
+Updating via Client
+-------------------
+
+With the client, instead of using ``endpoint.update()``, we will use the ``endpoint.update_from_config()`` method, which takes only the path to the config file:
+
+.. code-block:: python
+
+    endpoint.update_from_config("config_file.json")
+
+
+Updating via the CLI
+--------------------
+
+Updating the Endpoint with a config file via the CLI can be done using the `filename` option, or `f` for short:
```

Comment:
Do single backticks work as expected?

Reply:
I thought so; it "greyed out" the variables when I tried to build the html file. It doesn't show in GitHub preview though. I'll just double backtick just to be safe.

Reply:
Hm. I see a mix of the two. My main goal is to be consistent all accross.